packages/node_modules/@webex/plugin-meetings/README.md

```diff
@@ -326,124 +560,98 @@ webex.meetings.personalMeetingRoom.get().then((pmr) => {
 });
 ```
 
-##### Listing Active Meetings
-we want to sync our meetings collection with the server
+#### Usage of Webex Devices
+for details on how to use the devices, please see @webex/plugin-devices
 
+##### Leave a Meeting with Device
 ```js
-let existingMeetings;
-// Sync Meetings From Server
-webex.meetings.syncMeetings().then(() => {
-  // Existing meetings live in the meeting collection
-  existingMeetings = webex.meetings.meetingCollection.meetings;
-});
+let resourceId;
 
-```
-
-#### Member
+if (webex.devicemanager && webex.devicemanager._pairedDevice) {
```

Comment:
The underscore here denotes a private member. Should we remove this because we're encouraging its use?